tools/Customization/DevHome.Customization/CustomControls/DevDriveOptimizerCardBody.xaml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.Customization.CustomControls.DevDriveOptimizerCardBody"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:devEnvConverters="using:DevHome.Common.Environments.Converters"
+    mc:Ignorable="d">
+    <UserControl.Resources>
+        <devEnvConverters:CardStateColorToBrushConverter x:Key="CardStateColorToBrushConverter"/>
+        <devEnvConverters:CardStateToLocalizedTextConverter x:Key="CardStateToLocalizedTextConverter"/>
+    </UserControl.Resources>
+
+    <Grid
+            Style="{StaticResource CardBodyContainerStyle}">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+        <!--
+                Stackpanel containing name of cache to be optimized and a description.
+            -->
+        <StackPanel
+                Grid.Column="1"
+                Grid.ColumnSpan="2"
+                Style="{StaticResource CardBodyStackPanelStyle}">
+            <StackPanel Orientation="Vertical">
+                <Grid Margin="50 10 50 10">
+                    <TextBlock Margin="5" Grid.Column="0"
+                        Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}"
+                        Text="{x:Bind CacheToBeMoved, Mode=OneWay}"
+                        IsTextSelectionEnabled="True"/>
+                    <Grid HorizontalAlignment="Right">
+                        <FontIcon HorizontalAlignment="Left" Margin="-10" Glyph="&#xE946;" FontSize="12" FontFamily="{StaticResource SymbolThemeFontFamily}"/>
+                        <TextBlock HorizontalAlignment="Left" Margin="5 5 5 5" Grid.Column="1"
+                            Foreground="PowderBlue"
+                            Text="Suggestion"
+                            IsTextSelectionEnabled="True"/>
+                    </Grid>
+                </Grid>
+                <Line Margin="-5 10 -5 0" Stroke="Black" Stretch="Fill" X1="0" Y1="0" X2="1" Y2="0" StrokeThickness="1"/>

```

Comment:
`Line`?  Please use `Border` and wrap it around the element you want to put a border on.

Reply:
This is by design- [https://www.figma.com/file/AZ03I5bFPmrOpjVzFRjVn4/Dev-Home-Settings?type=design&node-id=2248-23987&mode=design&t=qtrgt0X2Dxli2fHU-4](https://www.figma.com/file/AZ03I5bFPmrOpjVzFRjVn4/Dev-Home-Settings?type=design&node-id=2248-23987&mode=design&t=qtrgt0X2Dxli2fHU-4)